css-flexbox/Overview.bs

```diff
@@ -1481,7 +1488,7 @@ Flexibility</h2>
 <h3 id='flex-property'>
 The 'flex' Shorthand</h3>
 
-	<pre class='propdef'>
+	<pre class='propdef shorthand'>
```

Comment:
It's because without that, bikeshed ends up generating "Animation type: discrete" (the default value for that line), but with "shorthand" added bikeshed generates "Animation type: see individual properties". That's what at least one of the other shorthands in this spec does.